### PR TITLE
Room insights initial commit

### PR DESCRIPTION
**Pull request by Jharv94**
# Add app insights into room, 204

## Change Proposal, Files added

- [x] APP_INSIGHTS environment variable inside appsettings.json
- [x] ApplicationInsights.Aspnetcore to room.csproj file
- [x] ApplicationInsights.PerfCounterCollector to room.csproj file to enable Live metrics
- [x] Application Insights Telemetry has been added to Configureservices in Startup
- [x] Health Checks for Room Api

- [x] Created RoomTelemetryInitializer file
- [x] Errorlogging to Room.Api appsettings.json
- [x] TelemetryInitializer to Room.Api Startup.cs configureservices method, this grabs: users, sessions, events
## reviewer

- @jamesgoldsmith8 
- @fredbelotte 

## contributing

this project is maintained under the terms of the [Contribution Guidelines][contributing].

[contributing]: https://github.com/fredbelotte/alpine/blob/master/.github/CONTRIBUTING.md 'contribution guidelines'

### CODE REVIEW

File: room/src/Revature.Room.Api/appsettings.json [hunk 1]
```diff
@@ -1,4 +1,5 @@
 {
+  "APPINSIGHTS_INSTRUMENTATIONKEY": "Putkeyhereplease",
```
Comment (fredbelotte) [inline room/src/Revature.Room.Api/appsettings.json]: should match placeholder patterns

File: manifest.docker.yaml [hunk 2]
```diff
@@ -31,7 +31,7 @@ services:
     image: postgres:alpine
     networks:
       - locally
-
+###############################################################################
```
Comment (fredbelotte) [inline manifest.docker.yaml]: remove break lines

File: room/src/Revature.Room.Api/appsettings.json [hunk 3]
```diff
@@ -1,4 +1,10 @@
 {
+  "APPINSIGHTS_INSTRUMENTATIONKEY": "Provide instrumentation key",
```
Comment (fredbelotte) [inline room/src/Revature.Room.Api/appsettings.json]: follow the guidelines

File: room/src/Revature.Room.Api/appsettings.json [hunk 4]
```diff
@@ -1,5 +1,5 @@
 {
-  "APPINSIGHTS_INSTRUMENTATIONKEY": "Provide instrumentation key",
+  "APPINSIGHTS_INSTRUMENTATIONKEY": "(provide instrumentation key)",
```
Comment (Jharv94) [inline room/src/Revature.Room.Api/appsettings.json]: Sorry I pushed this with a past tense naming convention... working on it! :)

**Pull request merged**